src/reactor/mod.rs

```diff
@@ -297,6 +306,21 @@ impl Inner {
     }
 }
 
+impl Handle {
+    fn inner(&self) -> Option<Arc<Inner>> {
+        match self.repr {
+            HandleRepr::Ptr { ref inner, .. } => inner.upgrade(),
+            HandleRepr::Global => global::reactor().and_then(|x| x.inner()),
+        }
+    }
+}
+
+impl Default for Handle {
+    fn default() -> Handle {
```

Comment:
Should the default handle point to the global reactor? For new users this might be bit confusing, and for users not using a global event loop this might point to the wrong event loop.

Would  `impl Handle { fn global() -> Handle }` (so we can call `Handle::global`) be a better approach?

Reply:
IIRC this was debated at some length on the RFC, but eventually this may not actually return a global handle in the sense that you'll be able to override the return value of `Handle::default` within a particular program scope.

Reply:
Ok.